### PR TITLE
initial commit from woody

### PR DESCRIPTION
**Pull request by thunderpoot**
look at my crappy rst

### CODE REVIEW

File: docs/source/lang.rst [hunk 1]
```diff
@@ -0,0 +1,1134 @@
+.. ___.                 .__
```
Comment (songb1rd) [inline docs/source/lang.rst]: Could this be placed in a codeblock so we don't have to be at the mercy of the font gods? (three backticks like a regular one)

Comment (thunderpoot) [reply docs/source/lang.rst]: it's a comment

Comment (songb1rd) [reply docs/source/lang.rst]: Ah, didn't notice. Its a pretty cool banner either way so its up to you to keep it in or not :D

**Comment by songb1rd** [comment]
:+1:

**Pull request merged**